src/stdlib/match_datadog_query.rs

```diff
@@ -120,7 +120,13 @@ impl Resolver for VrlFilter {}
 /// Implements `Filter`, which provides methods for matching against (in this case) VRL values.
 impl Filter<Value> for VrlFilter {
     fn exists(&self, field: Field) -> Box<dyn Matcher<Value>> {
-        let buf = lookup_field(&field);
+        let buf = match lookup_field(&field) {
+            Ok(path) => path,
+            Err(error) => {
+                error!(message = "Invalid query field", %error);
+                return Run::boxed(|_| false);
+            }
+        };
```

Comment:
This exact pattern is repeated several times here. Is there any way of DRYing it up other than a new macro?

Reply:
No longer relevant after https://github.com/vectordotdev/vrl/pull/1031/commits/551b0d21bc9521df08ef546fd557d7c3dc526573